include/ur_api.h

```diff
@@ -851,8 +852,8 @@ typedef enum ur_device_info_t {
     UR_DEVICE_INFO_PREFERRED_INTEROP_USER_SYNC = 74,            ///< [::ur_bool_t] prefer user synchronization when sharing object with
                                                                 ///< other API
     UR_DEVICE_INFO_PARENT_DEVICE = 75,                          ///< [::ur_device_handle_t] return parent device handle
-    UR_DEVICE_INFO_PARTITION_PROPERTIES = 76,                   ///< [::ur_device_partition_property_t[]] Returns an array of partition
-                                                                ///< types supported by the device
+    UR_DEVICE_INFO_PARTITION_PROPERTIES = 76,                   ///< [::ur_device_partition_t[]] Returns an array of partition types
```

Comment:
I think this type should be `ur_device_partition_property_t`

Reply:
Oh, okay. I don't think this should have `_PROPERTIES` for UR then, `UR_DEVICE_INFO_SUPPORTED_PARTITIONS` maybe?

Given these changes, I'd expect `UR_DEVICE_INFO_PARTITION_PROPERTIES` to be returning the array of `ur_device_partition_property_t`s used to create a paritioned device. There isn't a query for that at the moment, is there?

Reply:
The query that returns `ur_device_partition_property_t[]` is `UR_DEVICE_INFO_PARTITION_TYPE` which will return the partition properties passed when to the subdevice when it was created with `urDeviceParititon`. I find that these seem to be confusingly named in OpenCL and we've copied the style into UR. Maybe from your suggestion we could do the following:

`UR_DEVICE_INFO_PARTITION_PROPERTIES` -> `UR_DEVICE_INFO_SUPPORTED_PARTITIONS`
`UR_DEVICE_INFO_PARTITION_TYPE` -> `UR_DEVICE_INFO_PARTITION_PROPERTIES`

Reply:
I'm going to leave this as is for now, but I've created #552 to update the naming of these enumerations.